crates/bevy_color/src/spectral.rs

```diff
@@ -0,0 +1,279 @@
+use crate::{Alpha, Color, LinearRgba, Luminance, Xyza};
+use bevy_math::FloatExt;
+use std::ops::Mul;
+
+/// A color produced by monochromatic light (of a single wavelength)
```

Comment:
```suggestion
/// A color produced by monochromatic light (of a single wavelength).
```

Maybe(?). Also in all the fields and other places.